synapse/lib/stormtypes.py

```diff
@@ -2128,6 +2127,9 @@ async def wput(self, sha256, url, headers=None, params=None, method='PUT', ssl=T
         params = self.strify(params)
         headers = self.strify(headers)
 
+        if proxy is not None and not self.runt.allowed(('axon', 'proxy')):
+            raise s_exc.AuthDeny(mesg=s_exc.proxy_admin_mesg, user=self.runt.user.iden, username=self.runt.user.name)
```

Comment:
the proxy_admin_mesg value is no longer correct.

Can we use `confirm()` to generate the correct permissions error?